### ISSUE TITLE
function get_vipkeys returns keys in wrong format

### ISSUE THREAD

**Issue by PawelPodkalicki** [adshares/ads#93]
Below is response fot get_vipkeys funciton:
```
{
  "current_block_time": "1529588640",
  "previous_block_time": "1529588608",
  "tx": {
    "data": "15010000000000A5AB2B5B23A7002738367EC99A7BD2988720FF0824580219D2502226084E1BCE11A0B63438198FDC8747992E1D6B2C85A8BA84B2CC97D0394B7C34EB71FDDB7320E15A1011B527B8ED331ED2E1FAEDA72F13C79F2B541C2D12BEB40F17EE71DD76359A01"
  },
  "viphash": "23A7002738367EC99A7BD2988720FF0824580219D2502226084E1BCE11A0B634",
  "vipkeys": [
    "010073A5C92FA5142599B1C9863B43E026AFEFA6B57AEE8D189241C7F50C90BA",
    "51220200FC4CA38301AC2080ADA2BC08C4A94405E546B659BD2EB559C1520A55",
    "586336CD03005138AA57FAF5F7F2E67BDCA14F3CC377CD4C681B3F8B5A41DDDD",
    "590BD36C31250400BB6D774EA23DFB4D6510F04EFFA79FCA281C046CB39143B1",
    "01CB451D0919AFA90500BCEEFF93B281A5FB95B3F05922DC3D9B4BDE91FC8116",
    "C340068F778CA16F256406003A1969E7A8F418537CBCBEDB8FDC8E4C5B4D78C4",
    "D73970B9E2739055FF4E883E070064938E054C70E26F37E56474B20E78691ACE",
    "89A441018041FDFFAC2B224BE55C000020000000000000008100000000000000"
  ]
}
```
In test network were 7 vips.
```
1: 73A5C92FA5142599B1C9863B43E026AFEFA6B57AEE8D189241C7F50C90BA5122
2: FC4CA38301AC2080ADA2BC08C4A94405E546B659BD2EB559C1520A55586336CD
...
```
Note, that first key in response is 
```
0100
    73A5C92FA5142599B1C9863B43E026AFEFA6B57AEE8D189241C7F50C90BA
```
Seconds is
```
5122
    0200
        FC4CA38301AC2080ADA2BC08C4A94405E546B659BD2EB559C1520A55
```
This means that node id is included in key, which is wrong.

**Comment by PawelPodkalicki** [comment]
Response format was changed.

**Comment by PawelPodkalicki** [comment]
Fixed in 728a076f167db31e15681ca950b7ac1dd8ccd246
